src/site/stages/build/process-cms-exports/schemas/input/media-image.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable camelcase */
+
+module.exports = {
+  type: 'object',
+  properties: {
+    name: { $ref: 'GenericNestedString' },
+    thumbnail: {
+      type: 'array',
+      maxItems: 1,
+      items: {
+        type: 'object',
+        properties: {
+          alt: { type: ['string', 'null'] },
+          title: { type: ['string', 'null'] },
+          width: { type: 'number' },
+          height: { type: 'number' },
+          ...{ $ref: 'EntityReference' }.properties,
```

Comment:
I don't think this actually works. `$ref` is an internal JSON Schema thing. If you want to spread these, I'd recommend `require`ing the entity reference common schema, and spreading the `entityReferenceSchema.properties` that way.

Reply:
Ahh yeah `...entityReferenceSchema.properties` does the trick. Nice catch!